modules/view-images-in-your-environment.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
+:_module-type: PROCEDURE
+[id="view-images-in-your-environment_{context}"]
+= Viewing images in your environment
+
+[role="_abstract"]
+With {product-title} you can view the details for all container images in your clusters.
+
+.Procedure
+. Navigate to the RHACS portal and click *Vulnerability Management* from the left-hand navigation menu.
+. To view details for all the images in your cluster, on the *Vulnerability Management* view header, select *Images*.
```

Comment:
"To view details for all the images in your cluster, select *Images* on the *Vulnerability Management* view header." 

^ This might read a little clearer. Up to you!